### DISCUSSION TITLE
Why help is `Ctrl-b`?

### DISCUSSION THREAD

**Discussion by abitrolly** [yassinebridi/serpl#34] [Q&A]
Why not F1? 
Why not `alt-h` (easier to type and remember)?
Why not close help window with the same shortcuts?

**Comment by yassinebridi** [comment]
It's just a shortcut that works for me, `alt-h` or `ctrl-h` or `?` interfere with my setup.

You can customize it through your config file:
https://github.com/yassinebridi/serpl?tab=readme-ov-file#configuration

**Reply by abitrolly** [reply]
What is `alt-h`, `ctrl-h` and `?` in your setup? I am just curious.
